Entities/person.rb

```diff
@@ -27,4 +29,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] As mentioned earlier, kindly create a rental object using this method, it should have `date` and `book` as arguments and inside it should create an instance like this:

```
Rental.new(date, book, self)
```